Standards/scs-XXXX-vN-mandatory-and-supported-IaaS-services.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Mandatory and Supported IaaS Services
+type: Standard
+status: Draft
+track: IaaS
+---
+
+## Introduction
+
+To be SCS-compliant a CSP has to fulfill all SCS standards.
+Some of those standards are broad and consider ALL services on the IaaS-Layer.
+There exist many services on that layer and they need to be limited to have a clear scope for the standards and the Cloud Service Providers following them.
+For this purpose, this standard will establish lists for mandatory services that have to be present in a SCS cloud as well as supported services, which are considered by some standards and may be tested or even implemented in the reference implementation but are optional in a sense that their omission will not violate SCS conformance.
+
+## Motivation
+
+There are many OpenStack APIs and their corresponding services that can be deployed on the IaaS level.
+These services have differences in the quality of their implementation and liveness and some of them may be easily omitted when creating an IaaS deployment.
+To fulfill all SCS-provided standards only a subset of these APIs are required.
+Some more but not all remaining OpenStack APIs are also supported additionally by the SCS project and may be part of its reference implementation.
```

Comment:
I struggle a bit with

> ... and may be part of its reference implementation

I though recommended services **are** part of SCS reference implementation.

Reply:
I think we still need to discuss this. Some services were requested, but currently are not present in the reference implementation (Cloudkitty, Heat, Masakari, Skyline). Heat might still be excluded from the supported list.

Reply:
Only Masakari is not available in the reference implementation at the moment. The other things are deployable, even if we would like to do without some of them. Masakari is also deployable, but we have not yet built and tested the necessary images.

Reply:
Can this be marked as resolved?